src/retrace-server-cleanup.txt

```diff
@@ -13,13 +13,16 @@ DESCRIPTION
 -----------
 The tool collects different kinds of garbage created by Retrace server:
 
-* Deletes old tasks. The limit to proclaim task old can be changed
-in the configuration file (DeleteTaskAfter).
+* Deletes old tasks based on mtime. The limit to proclaim task 
```

Comment:
“The limit to proclaim task old can be changed in the configuration file” - this is now superfluous.

Reply:
Sure! I can delete that line.